shared/src/commonTest/kotlin/com/mbta/tid/mbta_app/viewModel/FavoritesViewModelTest.kt

```diff
@@ -388,16 +388,13 @@ internal class FavoritesViewModelTest : KoinTest {
                 awaitItem(),
             )
             // static data usually loads before realtime, but not always
-            assertEquals(
-                FavoritesViewModel.State(
-                    awaitingPredictionsAfterBackground = false,
-                    favorites = favorites.routeStopDirection,
-                    routeCardData = expectedRealtimeData,
-                    staticRouteCardData = expectedStaticData,
-                    loadedLocation = stop1.position,
-                ),
-                awaitItemSatisfying { it.routeCardData != null && it.staticRouteCardData != null },
-            )
+            awaitItemSatisfying {
+                (it.routeCardData == expectedRealtimeData &&
```

Comment:
non-blocking: What about just `awaitItemSatisfying { it == FavoritesViewModel.State(...) }`?